README.md

```diff
@@ -116,6 +116,138 @@ notifications):
 script: bundle exec rake run_about_yml_check
 ```
 
+
+## `.about.yml` cheat sheet
+The following attributes are currently stored in the `.about.yml` file and are used in one or more of the locations referenced above. Required attributes are marked with an asterisk, and field descriptions and examples are offered. Take a look at [Every Kid in a Park](https://github.com/18F/ekip-api/blob/master/.about.yml) or [Open Opportunities](https://github.com/18F/openopps-platform/blob/dev/.about.yml) for living examples of `.about.yml` in action.
```

Comment:
Still: "field descriptions and examples are offered" => "field descriptions and examples follow each attribute" per https://pages.18f.gov/content-guide/active-voice/.